app/routes/auth.js

```diff
@@ -164,6 +179,13 @@ router.post('/signup', (req, res, next) => {
 router.post('/verify', (req, res) => {
   const { email, confirmationId } = req.body;
 
+  if (!email || !confirmationId) {
```

Comment:
How would the client obtain the user's email from a confirmation code? Either:

 - generate unique confirmation IDs and remove the email requirement
 - send the email as a URL parameter to the client confirmation page

Reply:
I can do that, I just need an email template to send for now.

Reply:
Sending the email as a URL parameter? In that case, we can resolve this for now and I will create an issue on the client for this change.

Reply:
Yeah, I was thinking two URL parameters. Just something like `descartes.quantumstack.xyz/verify?email=aditya@example.com&confirmationId=abc`